lib/xboxrt/c_runtime/_alldiv.s

```diff
@@ -16,6 +16,16 @@
 
 // Modified for stdcall calling convention for use in nxdk
 
+// This creates a special symbol with the least significant bit set.
+// It signals to the linker that this code is SafeSEH-compatible.
+.text
+.def @feat.00;
+.scl  3; // IMAGE_SYM_CLASS_STATIC
+.type 0; // IMAGE_SYM_TYPE_NULL
+.endef
+.globl @feat.00
+.set @feat.00, 1
```

Comment:
Can't this be included from a `prelude.s` or be a macro or something?

Reply:
Done.

Reply:
Cheers. Much better.

*(Minor complaint: Filename could be a generic `prelude.s` if you still have to apply the macro afterwards anyway; but I don't care too much either)*

Reply:
Will merge if you don't plan to rename file (as in my minor complaint)
Although, I feel more strongly by now, because I don't want to add more and more files to contain re-usable macros, if we can just have one `prelude.s` which contains all the common 3-5 line macros.

Reply:
I've renamed the file now, but kept the `.s.inc` extension, I want to avoid people thinking that this is a normal assembly code file.